src/coordination/coordinator_state_machine.cpp

```diff
@@ -177,6 +299,13 @@ auto CoordinatorStateMachine::save_logical_snp_obj(snapshot &snapshot, ulong &ob
     auto ll = std::lock_guard{snapshots_lock_};
     auto entry = snapshots_.find(snapshot.get_last_log_idx());
     MG_ASSERT(entry != snapshots_.end());
+    if (durability_store_) {
+      auto snapshot_ptr = entry->second->snapshot_;
+      MG_ASSERT(durability_store_->Put(std::string{kSnapshotIdPrefix} + std::to_string(snapshot.get_last_log_idx()),
+                                       SerializeSnapshotCtxToJson(SnapshotCtx{snapshot_ptr, cluster_state}).dump()),
+                "Failed to store snapshot to disk.");
+    }
+    // TODO(antoniofilipovic): Check with Andi S. why we are not updating here also snapshot ptr
```

Comment:
What exactly do you mean? I was using example from NuRaft's page, nothing more than that

Reply:
fixed, as per call